### ISSUE TITLE
"explicitly specific" think-o in pb-copyable.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5266]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5266|
|Type|defect|
|Created|2011-09-21 00:44:47Z|

Pointed out by David Kao on the mailing list, pb-copyable.xhtml says _you must explicitly **specific** what remote classes will be accepted_ but means _you must explicitly **specify** what remote classes will be accepted_

Attachments:

* [specific.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/04c/04c7c383662a0037a20a3ae896d59d5c64565c8a/430a2a983cefb4d099a21544d1d25d92cd893c3f.patch) (672 bytes) - added by exarkun on 2011-09-21 00:45:34Z - Fix for the mistake

<details><summary>Searchable metadata</summary>

```
trac-id__5266 5266
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1316565887000000 1316565887000000
changetime__1316704680000000 1316704680000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Please merge with a misc file.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [32567]) Fix simple miswording in pb documentation

Author: exarkun
Reviewer: thijs
Fixes: [#5266](https://github.com/twisted/twisted/issues/5266)
